### PR TITLE
note `--user` args usage restriction

### PR DESCRIPTION
**Pull request by yuchengwu**
Signed-off-by: Yucheng Wu <wyc123wyc@gmail.com>

**- What I did**
note `--user` args usage restriction
fixes and closes: https://github.com/docker/docker.github.io/issues/8831

**- How I did it**
add additional explanation

**- How to verify it**
xref: https://github.com/moby/moby/issues/22323#issuecomment-215449380

**- Description for the changelog**

```
the user name specified by docker run `--user` args must be exist in the container
```

**- A picture of a cute animal (not mandatory but encouraged)**
peiqi
haha, his name is peiqi

### CODE REVIEW

File: docs/reference/run.md [hunk 1]
```diff
@@ -1605,6 +1605,7 @@ the `USER` instruction by passing the `-u` option.
     --user=[ user | user:group | uid | uid:gid | user:gid | uid:group ]
 
 > **Note:** if you pass a numeric uid, it must be in the range of 0-2147483647.
+> If you pass a user name, the user must be already exist in the container.
```
Comment (thaJeztah) [inline docs/reference/run.md]: Sorry for the long delay; think this PR dropped of my radar; I see a small grammatical error; `be already exist` should probably be `must already exist`, but perhaps we can simplify a bit;

```suggestion
> If you pass a username, the user must exist in the container.
```

Comment (yuchengwu) [reply docs/reference/run.md]: Thanks, it has been edited as suggested, please review again.

**Review by devdreamsolution — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM

Sorry, forgot about this one (I saw it needed squashing, so i did a quick rebase and squash)

thanks!

**Pull request merged**